src/Components/CreateImageWizard/steps/Oscap/index.tsx

```diff
@@ -88,6 +94,41 @@ const OscapStep = () => {
     dispatch(clearKernelAppend());
   };
 
+  if (process.env.IS_ON_PREMISE && !onPremOpenSCAPAvailable) {
```

Comment:
Maybe this should be taken up one level for two reasons:
- it takes a bit of time to execute the check for the packages so the warning shows for a moment even if the packages are available
- some of the queries to get the profile information are attempted and fail with errors if OpenSCAP is not available